test/fixtures/secrets/ejson-keys.yml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
```

Comment:
Could we re-use the ejson-keys fixture set for this? Both to avoid extra fixture sets, and to show the intended scary scenario explicitly in the test:

```ruby
ejson_fixtures.create_ejson_keys_secret # 1. something creates it normally
# 2. do a kubectl apply that adds keys or something, like a clueless user would
# 3. run a deploy like your test currently does, showing that we fixed the user's mistake
```